### PR TITLE
[MPS][BE] Code cleanup

### PR DESCRIPTION
**Pull request by malfet**
Various code cleanup in MPS operations:
 - Per @kulinseth suggestion move `mpsSupportsCumsum` to `MPSDevice.h` and rename it to
   `is_macos_13_or_newer()`
 - Move Ventura MPSGraph new operators to `MPSGraphVenturaOps.h` header
 - Use `LookupAs` and `CreateCachedGraphAs` to make code more compact
 - Formatting

### CODE REVIEW

File: aten/src/ATen/native/mps/OperationUtils.h [hunk 1]
```diff
@@ -263,6 +263,8 @@ struct MPSGraphCache
 
 };
 
+bool supportsVenturaOps();
```
Comment (kulinseth) [inline aten/src/ATen/native/mps/OperationUtils.h:266]: I was thinking of something like [this](https://github.com/pytorch/pytorch/pull/88532/files#diff-c7bb5bafa54cf2dfc86cafb769b80f8a8dfbf3342020f11b1b3fae507b56fef2R30)

**Comment by kulinseth** [comment]
Overall the change looks good to me.

**Review by kulinseth — APPROVED** [review]

File: aten/src/ATen/native/mps/OperationUtils.h (hunk 1 above)
Comment (malfet) [reply aten/src/ATen/native/mps/OperationUtils.h:266]: Thank you for the suggestion. Let me move it to MPSDevice (though really its not a function of device, but rather function of an OS, which is why I wanted to make it a static method)

**Pull request closed without merging**